jetty-core/jetty-server/src/main/config/modules/threadpool-all-virtual.mod

```diff
@@ -13,7 +13,16 @@ etc/jetty-threadpool-all-virtual.xml
 
 [ini-template]
 # tag::documentation[]
-## Platform threads name prefix.
+## Virtual threads name prefix.
 #jetty.threadPool.namePrefix=vtp<hashCode>
 
+## Maximum number of concurrent virtual threads.
+#jetty.threadPool.maxThreads=200
```

Comment:
200?!?!?!?  aren't you missing a few orders of magnitude here?   Nobody should be using virtual threads (and risking all sorts of bad behaviour) at 200 threads... maybe 2,000, but better 20,000
```suggestion
#jetty.threadPool.maxThreads=20000
```